fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -432,10 +437,56 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            log_Msg "---> No distortion correction"
+
+            log_Msg "---> FAKE WarpField"
+            ### faked warp image with three frames
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 ${WD}/WarpField.nii.gz
+            ${FSLDIR}/bin/fslmerge -t ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz
+
+            log_Msg "---> FAKE Jacobian Volume"
+            ###Jacobian Volume FAKED for Regular Fieldmaps (all ones) ###
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 -add 1 -bin ${WD}/Jacobian.nii.gz
+
+            log_Msg "---> FAKE spline interpolated image of scout"
+            # create a spline interpolated image of scout (distortion corrected in same space)
+            ${FSLDIR}/bin/imcp ${ScoutInputName} ${WD}/${ScoutInputFile}_undistorted
```

Comment:
And I don't see why we need to create a `${ScoutInputName}_undistorted` file either (which is in fact _distorted_).  It feels to me that this code is unnecessarily being driven by an attempt to "mimic" the SEFM correction code (which leads potentially to terrible confusion), rather than simply computing the necessary registration as its own de novo code, and then applying that registration.

Reply:
It looks like the `${ScoutInputName}_undistorted*` nomenclature is baked into the downstream FreeSurfer BBR.  My suggestion would be that we write "clean" code for the initial registration, and then at the end the "NONE" subsection we create any additional necessary files for the downstream code, with a very clear comment on why we are creating them.  And then possibly delete those fake/misnamed files within `${WD}` at the end of the DistortionCorrection script.

Reply:
For any files that are just copies, you might consider making symlinks instead, so that it ives a hint that they are fake.